wrappers/python/nntile/functions.py

```diff
@@ -76,14 +76,6 @@ def gemm_async(
         core_tensor.gemm_async_fp32_fast_tf32(
             alpha, trans_A, A, trans_B, B, beta, C, ndim, batch_ndim, redux
         )
-    elif type(A) is core_tensor.Tensor_fp32_fast_fp16:
```

Comment:
Lots of deleted lines in this file mean only one thing -- you messed up with versions.